src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -98,6 +104,22 @@ public void onPost(final GitHubTriggerEvent event) {
         if (Objects.isNull(job)) {
             return; // nothing to do
         }
+        if (useGitExcludedUsers) {
+            Set<String> excludedUsers = null;
+            if (job instanceof AbstractProject) {
+                SCM scm = ((AbstractProject<?, ?>) job).getScm();
+                if (scm instanceof GitSCM) {
+                    UserExclusion exclusions = ((GitSCM) scm).getExtensions().get(UserExclusion.class);
+                    if (exclusions != null) {
+                        excludedUsers = exclusions.getExcludedUsersNormalized();
+                    }
+                }
+            }
+
+            if (excludedUsers != null && excludedUsers.contains(event.getTriggeredByUser())) {
```

Comment:
Should be ignorecase

Reply:
It appears to me the polling is case sensitive as well:
https://github.com/jenkinsci/git-plugin/blob/a7ad6eb38a00397c4a656b53d99bfbad8ea3b2cc/src/main/java/hudson/plugins/git/extensions/impl/UserExclusion.java#L63
So if the webhooks checks the exclusion list in a case insensitive manner, there will be a counter-intuitive difference...

Reply:
gitscm extension is for git users, github users case insensitive you do comparison in terms of GH functionality it should be ignorecase

Reply:
Done